tests/ui/specs/VirtualList/VirtualGridList/With5-way/VirtualGridList-GT-28454-specs.js

```diff
@@ -0,0 +1,47 @@
+const {expectFocusedItem} = require('../../VirtualList-utils');
+const Page = require('../VirtualGridListPage');
+
+describe('Navigate with 5-way', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should navigate from and to last item via 5-way [GT-28454]', function () {
+		// Adjust the dataSize to make sure 3 rows of items show with *only* 1 (one) item on the last row.
+		Page.inputMinWidth.moveTo();
```

Comment:
Could you add more description about below step?
//  minWidth > 400 , minHeight > 800 ,